packages/webapp/src/webview/types.ts

```diff
@@ -15,4 +16,6 @@ export interface AppState {
     guideFeedback: null | boolean;
     selectedProductFilters: string[];
     selectedComponentFilters: string[];
+    updatedGuidedAnswerTrees: GuidedAnswerTreeSearchHit[];
```

Comment:
As mentioned, I'd use `guidedAnswerTreeSearchResult.trees` instead of a new property

Reply:
Changed, thanks!